src/en/getting-started.md

```diff
@@ -49,52 +57,40 @@ particular cloud provider. This is done by generating and editing the following
 ## Windows
     %LOCALAPPDATA%/Juju
```

Comment:
yaml file ends up here:

C:\Users\{user}\AppData\Roaming\Juju\environments.yaml

where

%APPDATA% = C:\Users\{username}\AppData\Roaming

Not LOCALAPPDATA and not .juju

Also assuming Windows is installed on the C: drive.